glossary.md

```diff
@@ -45,6 +53,10 @@ At the national level, policy and legislation (the law) are usually separate. Th
 
 In the Standard the word 'policy' refers to policy created and adopted by public organizations such as governments and municipalities.
 
+## Policy makers
+
+Politicians or senior civil servants that have decision making power or influence over which policies the codebase is aiming to implement.
```

Comment:
I describe this group as the people who decide what the outcomes should be of a given thing and its priority, as well as how this is codified in legal code.

***

Currently, this describes mostly a power relationship, not the roles these people have.
(Perhaps highlighting how we have less understanding of this target group)

***

My suggestion:

```suggestion
People tasked with formulating the outcomes, describing the goals, prioritising as well as developing the policies and legal code for a codebase.

For example:

* Senior civil servants
* Policy makers
* Policy advisors
* Politicians
* Political advisors
```